### DISCUSSION TITLE
Crowdsec in LXC

### DISCUSSION THREAD

**Discussion by espinardo** [tteck/Proxmox#1061] [Help Needed] [answered]
Greetings to all

I try to deploy the Crowdsec LXC on a fresh installation, but I got the following message:
Can't Install on Proxmox.

Later, I upgraded Proxmox using the "Post Install" tteck helper, but still got the same error.

I have opened the crowdsec.sh file and I see:

if command -v pveversion >/dev/null 2>&1; then echo -e "⚠️  Can't Install on Proxmox "; exit; fi
while true; do
    read -p "This will Install ${APP} on $hostname. Proceed(y/n)?" yn
    case $yn in
    [Yy]*) break ;;
    [Nn]*) exit ;;
    *) echo "Please answer yes or no." ;;
    esac
done

Would it be necessary to install some more dependency to be able to enjoy Crowdsec through the TTECK helper?

Thank you very much in advance!!

**Comment by tteck** [comment, marked as answer]
"**To Install CrowdSec, ⚠️ run the command below in the LXC console.**"  You can't run the command in the Proxmox shell.

**Reply by tteck** [reply]
> I try to deploy the Crowdsec LXC on a fresh installation

This script is not designed to create a new LXC container. Rather, it should be executed within an existing LXC console in order to add Crowdsec.
